scripts/test-runner.js

```diff
@@ -344,10 +427,14 @@ class ProgramArgs {
   debugNames: boolean;
   verbose: boolean;
   filter: string;
-  constructor(debugNames: boolean, verbose: boolean, filter: string) {
+  repl: string;
```

Comment:
Can you add comments for the meaning of these two flags?

Reply:
:+1: